data/presets/shop/water_sports.json

```diff
@@ -7,5 +7,8 @@
     "tags": {
         "shop": "water_sports"
     },
+    "terms": [
+        "swimwear"
```

Comment:
Would a "dive shop" be part of this, or is there a separate tag for that?

Reply:
On an unrelated note, I was listening to this right now: https://youtu.be/F7-NR6cClTw?t=1912

Reply:
I will remove swimwear, apparently it is a different tag: 

> [shop](https://wiki.openstreetmap.org/wiki/Key:shop)=[clothes](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dclothes) + [clothes](https://wiki.openstreetmap.org/wiki/Key:clothes)=[swimwear](https://wiki.openstreetmap.org/wiki/Tag:clothes%3Dswimwear) – a shop sells swimming suits